src/hatti/views/table.cljs

```diff
@@ -416,44 +416,52 @@
    each of whose value is a function of the form (fn [event args]) as described
    in the SlickGrid documentation for event handlers.
    https://github.com/mleibman/SlickGrid/wiki/Getting-Started"
-  (reify
-    om/IRenderState
-    (render-state [_ _]
-      (let [no-data? (empty? (get-in app-state [:data]))
-            {:keys [num_of_submissions] :as dataset-info}
-            (:dataset-info app-state)
-            with-info #(merge % {:dataset-info dataset-info})]
-        (html
-         [:div.table-view
-          (om/build submission-view
-                    (with-info (get-in app-state
-                                       [:table-page :submission-clicked]))
-                    {:opts (merge (select-keys opts #{:delete-record! :role})
-                                  {:view :table})})
-          (om/build table-header app-state)
-          [:div {:id table-id :class "slickgrid"}
-           (if (and no-data? (zero? num_of_submissions))
-             [:span {:class "empty-state"} "No data"]
-             [:span
-              [:i.fa.fa-spinner.fa-pulse] "Loading..."])]])))
-    om/IDidMount
-    (did-mount [_]
-      (let [data (get-in app-state [:data])]
-        (when-let [[grid dataview]
-                   (init-grid! data owner slick-grid-event-handlers)]
-          (handle-table-events app-state grid dataview))))
-    om/IWillReceiveProps
-    (will-receive-props [_ next-props]
-      "will-recieve-props resets slickgrid data if the table data has changed."
-      (let [old-data (get-in (om/get-props owner) [:data])
-            new-data (get-in next-props [:data])
-            {:keys [grid dataview]} (om/get-state owner)]
-        (when (not= old-data new-data)
-          (if (empty? old-data)
+  (let [active? (-> app-state
+                    :views
+                    :active
+                    set
+                    (contains? :table))]
```

Comment:
Thought so too, but was surprised not to find it in Hatti. It's in Chimera now, right?